ldap/servers/slapd/back-ldbm/db-mdb/mdb_config.c

```diff
@@ -544,7 +544,38 @@ dbmdb_ctx_t_serial_lock_set(void *arg,
     return LDAP_SUCCESS;
 }
 
+static void *
```

Comment:
I wonder if  backend.py should not be modified too:  maybe nsslapd-cache-autosize should be moved from bdb only to global

Reply:
yeah definitely and the UI will probably need an update as well

Reply:
BTW I'm almost done with the PR, just have to handle the in-memory only behavior